sdcm/utils/common.py

```diff
@@ -756,11 +758,11 @@ def get_instances(region):
 
 def clean_instances_aws(tags_dict, dry_run=False):
     """Remove all instances with specific tags in AWS."""
-    # pylint: disable=too-many-locals
+    # pylint: disable=too-many-locals,import-outside-toplevel
     assert tags_dict, "tags_dict not provided (can't clean all instances)"
     aws_instances = list_instances_aws(tags_dict=tags_dict, group_as_region=True)
-    from sdcm.argus_test_run import ArgusTestRun  # pylint: disable=import-outside-toplevel
-    argus_run = ArgusTestRun.get()
+    creds = KeyStore().get_argus_rest_credentials()
+    argus_client = ArgusSCTClient(run_id=tags_dict["TestId"], auth_token=creds["token"], base_url=creds["baseUrl"])
```

Comment:
these lines are repeating in multiple places. Can you hide it in some function? Only `test_id` looks customizable here.

Reply:
Done